src/routes/safe/components/Balances/SendModal/screens/ReviewCollectible/index.tsx

```diff
@@ -90,7 +60,7 @@ const ReviewCollectible = ({ onClose, onPrev, tx }: Props): React.ReactElement =
       try {
         const txData = await generateERC721TransferTxData(tx, safeAddress)
```

Comment:
This is a bit confusingly named after renaming the state to `txData`.

Reply:
why?

Reply:
The state is called `txData` as well.